projects/packages/search/src/dashboard/components/pages/sections/plan-usage-section.jsx

```diff
@@ -108,18 +112,60 @@ const UpgradeTrigger = ( { type, ctaCallback } ) => {
 	);
 };
 
-const UsageMeters = ( { usageInfo } ) => {
+const UsageMeters = ( { usageInfo, isJustUpgraded } ) => {
+	const [ currentTooltipIndex, setCurrentTooltipIndex ] = useState( isJustUpgraded ? 1 : 0 );
+	const goToNext = useCallback( () => setCurrentTooltipIndex( idx => idx + 1 ), [
+		setCurrentTooltipIndex,
+	] );
+
+	const tooltips = {
```

Comment:
What do you think about moving most of the tooltip definition outside the render function? It seems excessive to constantly re-instantiate this object when it could be memorized for changes to `currentTooltipIndex` and `goToNext`.